src/Service/ConsoleOptionsReader.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the box project.
+ *
+ * (c) Anatoliy Melnikov <5785276@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Aeliot\PhpCsFixerBaseline\Service;
+
+use PhpCsFixer\Config;
+use PhpCsFixer\Finder;
+
+class ConsoleOptionsReader
+{
+    private string $rootDirectory;
+
+    /** @var array<string, string>  */
+    private array $option;
+
+    private static ?self $instance = null;
+
+    public function __construct()
+    {
+        self::$instance = $this;
+        $this->option = getopt('ab:c:d:f:w:', ['absolute', 'baseline:', 'config:', 'dir:', 'finder:', 'workdir:']);
+        $this->rootDirectory = ($this->option['d'] ?? $this->option['dir']) ?? '';
```

Comment:
Check conflict when both options are passed. Consider using of the method `$this->getOptionValue()`